docs/sources/features/datasources/elasticsearch.md

```diff
@@ -39,7 +39,7 @@ All requests will be made from the browser to Grafana backend/server which in tu
 
 ### Browser (Direct) access
 
-All requests will be made from the browser directly to the data source and may be subject to Cross-Origin Resource Sharing (CORS) requirements. The URL needs to be accessible from the browser if you select this access mode.
+All requests will be made from the browser directly to the data source and may be subjected to Cross-Origin Resource Sharing (CORS) requirements. The URL needs to be accessible from the browser if you select this access mode.
```

Comment:
Please revert this edit, it's wrong:
```suggestion
All requests will be made from the browser directly to the data source and may be subject to Cross-Origin Resource Sharing (CORS) requirements. The URL needs to be accessible from the browser if you select this access mode.
```

Reply:
i think the change is correct, it should be subjected

Reply:
I don't think so, because "subject to Cross-Origin Resource Sharing (CORS) requirements" makes sense in this sentence. See [this example](https://www.law.cornell.edu/cfr/text/40/63.11081).

Reply:
@aknuds1 is correct. Please commit the change.

Reply:
FYI, if it was almost any other verb except "subjected", then you would be correct. English is weird.